pkg/storage/etcd3/watcher_test.go

```diff
@@ -123,7 +123,7 @@ func TestDeleteTriggerWatch(t *testing.T) {
 	if err := store.Delete(ctx, key, &api.Pod{}, nil); err != nil {
 		t.Fatalf("Delete failed: %v", err)
 	}
-	testCheckResult(t, 0, watch.Deleted, w, storedObj)
+	testCheckResult(t, 0, watch.Deleted, w, nil)
```

Comment:
can we still check the expected storeObject? we probably need to change the rev of the storeObj? Or if we do not care about the triggered result, we should explicitly express that when calling the testCheckResult func.